imports/message-read-receipt/server/lib/ReadReceipt.js

```diff
@@ -84,7 +84,11 @@ export const ReadReceipt = {
 			}
 
 			try {
-				rawReadReceipts.insertMany(receipts);
+				rawReadReceipts.insertMany(receipts, (err) => {
```

Comment:
I think it may be confusing having a `try/catch` that doesn't catch an error, so since [insertMany](https://mongodb.github.io/node-mongodb-native/3.4/api/Collection.html#insertMany) also returns a `Promise`, I suggest using `await` (instead of the error callback), so the `try/catch` block will work as expected.. to be able to use the `await` you'll need to change the function to `async`, so:
```javascript
async  storeReadReceipts(messages, roomId, userId, extraData = {}) {

...

			try {
				await rawReadReceipts.insertMany(receipts);
			} catch (e) {
				console.error('Error inserting read receipts per user');
			}

...

}
```

what do you think?

Reply:
Yes I totally agree, it would be a better fix for this.
I'll edit the code by your suggestion :)